scripts/release.js

```diff
@@ -97,7 +97,7 @@ const checkNPMDepsInstall = () => {
   }
   process.chdir(path.join(__dirname, '../../dependency-check-app'));
   spawnOrFail('npm', ['init -y']);
-  spawnOrFail('npm', ['install react@18 react-dom@18']);
```

Comment:
how is this related to version bump

Reply:
This was added during preflight. The release script failed due to incompatible React version. Now the latest React version is 19, but we only set 17, and 18 as peerDependency.